packages/liquidator/mocha-test/liquidationStrategy.js

```diff
@@ -0,0 +1,284 @@
+const web3 = require("web3");
+const assert = require("assert");
+const Events = require("events");
```

Comment:
what does this do exactly?

Reply:
Was using this in a test using it but removed it. Will add the check back in to show an example of how to test for logs.